Client/qtTeamTalk/mainwindow.cpp

```diff
@@ -822,6 +821,8 @@ void MainWindow::loadSettings()
 
     slotUpdateUI();

```

Comment:
This line should be at the bottom of the function.

Reply:
If I move this to the bottom, this will be after eventual display of server list, so it's better to keep it here, right?

Reply:
It wouldn't matter because slotClientConnect() calls slotUpdateUI() anyway.